LayoutTests/imported/w3c/web-platform-tests/fullscreen/api/document-exit-fullscreen-nested-shadow-dom-expected.txt

```diff
@@ -1,17 +1,11 @@
-CONSOLE MESSAGE: Unhandled Promise Rejection: Error: assert_equals: expected Element node <div id="outer">
-      <slot></slot>
-    </div> but got null
+CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: Type error
```

Comment:
Do we know why we're still failing?

Reply:
It's throwing one of the errors here: https://searchfox.org/wubkat/rev/919c3b81c0474dfbe49620a512bdfad1c0f23d0f/Source/WebCore/dom/FullscreenManager.cpp#76-77

I have yet to figure out why.